updates.html

```diff
@@ -5,7 +5,7 @@
 ---
 {% include header.html %}
 <section>
-  <div class="jumbotron jumbotron-xl" style="background-image: url('//crds-cms-uploads.imgix.net/content/images/crossroads-church-updates2.jpg');" data-optimize-bg-img>
+  <div class="jumbotron jumbotron-xl" style="background-image: url('//crds-cms-uploads.imgix.net/content/images/crossroads-church-updates2.jpg?auto=format,compress&w=10');" data-optimize-bg-img>
```

Comment:
The way this bg-img comes in is weird. Not sure if it's bc the width is so small or something else. The transition is pretty jarring though.